src/zkevm_specs/evm/instruction.py

```diff
@@ -290,6 +290,15 @@ def compare(self, lhs: Expression, rhs: Expression, n_bytes: int) -> Tuple[FQ, F
         assert rhs.expr().n < 256**n_bytes, f"rhs {rhs} exceeds the range of {n_bytes} bytes"
         return FQ(lhs.expr().n < rhs.expr().n), FQ(lhs.expr().n == rhs.expr().n)
 
+    def compare_word(self, lhs: RLC, rhs: RLC) -> Tuple[FQ, FQ]:
```

Comment:
Can we doc this?
```suggestion
    def compare_word(self, lhs: RLC, rhs: RLC) -> Tuple[FQ, FQ]:
        """
        The first output value is 1 if the left-hand side is strictly smaller, 0 otherwise.
        The second output value is 1 if the left-hand side is equal to the right-hand side, 0 otherwise.
        """
```

Reply:
Added the doc